Orange/widgets/regression/owsgdregression.py

```diff
@@ -180,6 +162,22 @@ def _on_func_changed(self):
         for spin, enabled in zip(self._func_params, mask):
             spin.setEnabled(enabled)
 
+    def send_report(self):
+        items = OrderedDict()
+        items['Loss function'] = self.LOSS_FUNCTIONS[self.loss_function]
+        if self.loss_function != self.SqLoss:
+            items['Loss function'] +=  ", ε={}".format(self.epsilon)
+        items['Penalty'] = self.PENALTIES[self.penalty_type]
+        if self.penalty_type == self.ElasticNet:
+            items['Penalty'] += ": L1 : L2 = {} : {}".format(
+                                self.l1_ratio, 1.0 - self.l1_ratio)
+        items['Penalty'] = items['Penalty'] + ', α={}'.format(self.alpha)
+        items['Learning rate'] = ", η0={}".format(self.eta0)
```

Comment:
The type of learning rate (Constant of Inverse scaling) is missing. This code now produces
```
Learning rate: , η0=0.01
```

(While fixing this, also enclose 0 in `<sub>`.)